src/Components/PaperEditor/TranscriptsContainer/TranscriptTabPane/TranscriptTabContent/index.js

```diff
@@ -63,6 +63,9 @@ const TranscriptTabContent = (props) => {
             return { ...doc.data(), id: doc.id, display: true };
           });
           const lbls = tempLabels ? tempLabels : []; // to remove once a bug is fixed
+          tempLabels.sort((a, b) => {
```

Comment:
Is this a feature for the PR? Jw because we're not displaying when a label was created, and so that may not be as intuitive as say, alphabetical by label title.

Reply:
I wanted it so that when a new label was added, it went to the bottom of the list, previously it would save the labels seemingly in a random order… maybe @nganthi  has some good thoughts about how these should be ordered? For me it just felt intuitive that new labels are added to the bottom

Reply:
Totes makes sense, sounds like this shouldn't be a blocker for merging :)

Reply:
I've actually removed this sorting now and seems to work fine - but let me know what you think!